src/site/content/en/lighthouse-pwa/codelab-apple-touch-icon/index.md

```diff
@@ -55,7 +55,7 @@ the example app to your iOS device's home screen when an Apple touch icon hasn't
 1. Open Safari for iOS.
 1. Open the URL of your example app. This is the URL like `https://example.glitch.me` that you
    noted earlier.
-1. Tap **Share** <img style="height:1.2em;vertical-align:top;" src="share.png"/>  >
+1. Tap **Share** {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/79A8XsdBaHeLGY5Wdm1J.png", alt="Apple Share Button", width="15", height="20", style="vertical-align:top;" %}  >
```

Comment:
It seems like they were purposefully trying to make this image larger than its default height. 1.2em would usually be 19.2px

Reply:
fixed, i set the width and height in em in the style too so it doesnt look squished